package.json

```diff
@@ -34,7 +35,7 @@
     "release:minor": "changelog -m && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version minor && git push origin && git push origin --tags",
     "release:major": "changelog -M && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version major && git push origin && git push origin --tags",
     "start": "nodemon --ignore test/",
-    "test": "NODE_ENV=test istanbul cover _mocha -- test --require test/setup.js --recursive --timeout 30000",
+    "test": "yarn lint && NODE_ENV=test istanbul cover _mocha -- test --require test/setup.js --recursive --timeout 30000",
```

Comment:
Can you take the `yarn lint` out of here? I think it's best to have your scripts separate and then if you want to run tests and lint locally just run `yarn test && yarn lint`?

Reply:
Good point. I'll update my workflow to do that instead.